crates/runtime/src/accelerated_table.rs

```diff
@@ -86,15 +90,56 @@ fn validate_refresh_period(refresh: &Refresh, dataset: &str) {
     }
 }
 
-#[allow(clippy::too_many_arguments)]
+pub struct Options {
```

Comment:
👍. Just a thought in terms of style Spice may want to have.

I've seen builder pattern is quite popular, in the codebase, there is subscriber builder, dataframe builder, etc.

Is AccelerationTableBuilder with `with_retention` etc is more idiomatic?

Reply:
Yeah, good point. I'll switch it over to the builder.